### DISCUSSION TITLE
[request] Change owner in change attributes

### DISCUSSION THREAD

**Discussion by pwn0r** [doublecmd/doublecmd#1088] [Suggestions]
### Description

Currently 1.10 doublecmd on Ubuntu 22.04. This though affects all ubuntu/debian versions that I have used. 

Please add an option to change an owner of a file/folder in the change attributes window.  This window is a big time saver per se, as it allows to properly change attributes for files the current user has no write rights.  Unlike *normal* properties dialog, root permissions will be prompted. 

However it does not allow to change ownership.

**Comment by alexx2000** [comment]
It will be in the _normal_ properties dialog, plus recursive flag.

**Comment by pwn0r** [comment]
> 

OK, if normal properties actually ask for elevation. For now only change attributes prompts for this.

**Comment by alexx2000** [comment]
It will ask.
